app-commons/src/main/scala/org/bitcoins/commons/jsonmodels/bitcoind/WalletResult.scala

```diff
@@ -437,6 +470,20 @@ object AddressInfoResultPostV21 {
   }
 }
 
+case class listDescriptorsResult(
```

Comment:
Class names follow camel casing format. 

Class names should look like `ListDescriptorsResult` 

where as a `val` name is 

```scala
val descriptorResult = ListDescriptorsResult()
```